test/fixtures/cldr/main/en/dateFields.json

```diff
@@ -0,0 +1,3 @@
+{
+  "FIXME": "FIXME"
+}
```

Comment:
An easy way is to get this file is to:

    cp external/cldr-data/main/en/dateFields.json test/fixtures/cldr/main/en/

If you don't have `external/cldr-data/main/en/dateFields.json`, see https://github.com/jquery/globalize#install-development-external-dependencies